examples/lsc/mag240m/gnn.py

```diff
@@ -234,9 +237,15 @@ def configure_optimizers(self):
                     num_layers=len(args.sizes), dropout=args.dropout)
         print(f'#Params {sum([p.numel() for p in model.parameters()])}')
         checkpoint_callback = ModelCheckpoint(monitor='val_acc', mode = 'max', save_top_k=1)
-        trainer = Trainer(gpus=args.device, max_epochs=args.epochs,
-                          callbacks=[checkpoint_callback],
-                          default_root_dir=f'logs/{args.model}')
+        try:
```

Comment:
Can we use the version instead of try, except?

Reply:
yes will do